app/views/admin/appointment_summaries/index.html.erb

```diff
@@ -47,6 +48,7 @@
         <td><%= summary.date_of_appointment.to_s(:gov_uk) %></td>
         <td><%= summary.requested_digital ? 'Yes' : 'No' %></td>
         <td><%= summary.email %></td>
+        <td><%= link_to 'Edit email', edit_admin_appointment_summary_path(summary), class: 't-edit-email' %></td>
```

Comment:
Do we need to put a conditional in here to only show this button if the customer chose a digital appointment summary?